lib/interface_quda.cpp

```diff
@@ -2086,8 +2142,11 @@ quda::cudaGaugeField *checkGauge(QudaInvertParam *param)
                                     param->cuda_prec_refinement_sloppy, param->cuda_prec_eigensolver};
       QudaReconstructType recon[4] = {gaugeSloppy->Reconstruct(), gaugePrecondition->Reconstruct(),
                                       gaugeRefinement->Reconstruct(), gaugeEigensolver->Reconstruct()};
+
+      printfQuda("QudaInvertParam has different precisions than loadGaugeQuda, so recomputing sloppy fields\n");
```

Comment:
I like the addition of this for logging reasons---I'd put this in a `logQuda(QUDA_VERBOSE, [...]);`, I think.